content/en/docs/ops/deployment/performance-and-scalability/index.md

```diff
@@ -115,8 +115,8 @@ section, we will present both `jitter` and `non-jitter` performance measurements
 
 Inside the mesh, a request traverses the client-side proxy and then the server-side
 proxy. In the default configuration of Istio {{< istio_release_name >}} (i.e. Istio with telemetry v2),
-the two proxies add about 2.76 ms and 2.88 ms to the 90th and 99th percentile latency, respectively, over the baseline data plane latency.
-After enabling `jitter`, those numbers reduced to 1.72 ms and 1.91 ms, respectively. We obtained these results using the [Istio benchmarks](https://github.com/istio/tools/tree/{{< source_branch_name >}}/perf/benchmark)
+the two proxies add about 2.65 ms and 2.91 ms to the 90th and 99th percentile latency, respectively, over the baseline data plane latency.
```

Comment:
Did we have a regression or are we documenting a difference caused by test variance?

Reply:
we are documenting a difference caused by test variance